t/opentracing.t

```diff
@@ -0,0 +1,41 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+$ENV{OPENTRACING_TRACER} ||= 'jaeger';
+
+repeat_each(1);
+run_tests();
+
+
+__DATA__
+=== TEST 1: OpenTracing
+Request passing through APIcast should publish OpenTracing info.
+--- configuration
+    {
+        "services": [
+        {
+            "proxy": {
+            "policy_chain": [
+            { "name": "apicast.policy.upstream",
+                "configuration":
+                {
+                    "rules": [ { "regex": "/", "url": "http://echo" } ]
+                }
+            }
+            ]
+        }
+        }
+        ]
+    }
+--- request
+GET /a_path?
+--- response_body eval
+qr/uber-trace-id: /
+--- error_code: 200
+--- no_error_log
+[error]
+--- udp_listen: 6831
```

Comment:
That's openresty? cool.

Reply:
Thats the Test::Nginx testing framework: http://search.cpan.org/~agent/Test-Nginx/lib/Test/Nginx/Socket.pm#udp_listen